common-items.lic

```diff
@@ -253,7 +253,9 @@ module DRCI
   end
 
   def get_item_list(container)
-    container_contents = DRC.bput("look in #{container}",'In the .* you see (.*)\.').match(/In the .* you see (?:a|an|some) (?<items>.*)\./)[:items].split(/(?:, | and )?(?:a|an|some) /)
+    container_contents = DRC.bput("look in #{container}", 'In the .* you see (.*)\.', 'There is nothing')
```

Comment:
💡 I think you could also remove `container_contents =` since the var is never used; the function just automatically returns the last evaluation